lab_3_generate_by_ngrams/main.py

```diff
@@ -309,6 +478,19 @@ def get_next_token(self, sequence: tuple[int, ...]) -> list[tuple[int, float]] |
 
         In case of corrupt input arguments or methods used return None.
         """
+        if not isinstance(sequence, tuple) or not sequence or self._model is None:
+            return None
+        probs = self._model.generate_next_token(sequence)
+        if probs is None:
+            return None
+        if check_dict(probs, int, float, True):
```

Comment:
what???????? it is the third lab, look at this code, you made these mistakes only in the first week of the first lab. Look one more time, please, Elizaveta